content/root/quick-start-guide.textile

```diff
@@ -288,6 +304,17 @@ realtime
     }
 });
 
+bc[go]. // Instantiate the Ably client. Calling ably.WithAutoConnect(false) is optional
+// but ensures that you don't miss state changes.
+client, err := ably.NewRealtime(ably.WithKey("{{API_KEY}}"), ably.WithAutoConnect(false))
+if err != nil {
+  panic(err)
```

Comment:
Go uses tabs for indentation (see [here](https://pkg.go.dev/cmd/gofmt#pkg-overview)), so I think we should use a tab here:

```go
if err != nil {
	panic(err)
}
```

Reply:
Thanks @lmars - fixed all the alignment issues.